### ISSUE TITLE
[Feature] Automatically read the node and export all widgets?

### ISSUE THREAD

**Issue by ghostsquad** [chrisgoringe/cg-quicknodes#15]
This is basically a duplicate of this issue:
https://github.com/rgthree/rgthree-comfy/issues/96

(Not sure if this should live here, or in rgthree, given the intersection of functionality)

Anyways, basically, this is a feature request to hook up `Read Widget` to a node (similar to that of the `Fast Muter` and `Fast Bypasser` nodes), and simply _read_ all the widgets from that node (that are not already exported), and export them.

Thanks!

**Comment by chrisgoringe** [comment]
Export them how/in what format?

**Comment by ghostsquad** [comment]
currently trying to simplify this:

![image](https://github.com/chrisgoringe/cg-quicknodes/assets/903488/e037cb78-ee6a-4f9e-a93c-472b621eb509)

**Comment by chrisgoringe** [comment]
So the suggestion (I think?) is a node that, given the id of another node, creates a set of outputs that echo the widgets on the other node.

Is that right?

**Comment by ghostsquad** [comment]
Yep! that's basically it. Extra credit to match the widget type, but defaulting to "text" would probably also be ok for a first iteration.

**Comment by chrisgoringe** [comment]
Ok, Will look at that when I get back from travelling :)
